flow/src/org/labkey/flow/controllers/executescript/chooseRunsToAnalyze.jsp

```diff
@@ -58,28 +58,28 @@
     }
 </style>
 <labkey:errors/>
-<labkey:form method="POST" action="<%=new ActionURL(AnalysisScriptController.ChooseRunsToAnalyzeAction.class, getContainer())%>">
+<labkey:form name="analyzeRuns" method="POST" action="<%=new ActionURL(AnalysisScriptController.ChooseRunsToAnalyzeAction.class, getContainer())%>">
     <table class="lk-fields-table">
         <tr><td>Analysis script to use:</td>
-            <td><select name="scriptId" onchange="this.form.submit()">
+            <td><select id="scriptId" name="scriptId">

```

Comment:
We really need to add <labkey:select> to our taglib.

Reply:
Not a tag, but you can use `SelectBuilder` like this `<%=select().id().name().addOptions(Map<?, String>).selected()%>`